### PR TITLE
add a hint about the disabling completion descriptions to document

### PR DESCRIPTION
**Pull request by Shihta**
For the #1894

### CODE REVIEW

File: shell_completions.md [hunk 1]
```diff
@@ -173,6 +173,21 @@ backend        frontend       database
 Note that without declaring `rc` as an alias, the completion algorithm would not know to show the list of
 replication controllers following `rc`.
 
+#### Disable completion descriptions
+
+If you don't want to show descriptions in the completion, you can add `--no-descriptions` to disable it, like:
```
Comment (marckhouzam) [inline shell_completions.md:178]:
```suggestion
If you don't want to show descriptions in the completions, you can add `--no-descriptions` to the default `completion` command to disable them, like:
```

File: shell_completions.md [hunk 2]
```diff
@@ -173,6 +173,21 @@ backend        frontend       database
 Note that without declaring `rc` as an alias, the completion algorithm would not know to show the list of
 replication controllers following `rc`.
 
+#### Disable completion descriptions
```
Comment (marckhouzam) [inline shell_completions.md:176]: Could you move this new section under the existing "Descriptions for completions" section?

**Review by marckhouzam — CHANGES_REQUESTED** [review]
Thanks for the contribution!
Small tweek and we can get this in.

**Comment by Shihta** [comment]
Both are fixed

**Review by marckhouzam — APPROVED** [review]
Thanks!

**Pull request merged**